### DISCUSSION TITLE
Alternatives to a recurring subscription?

### DISCUSSION THREAD

**Discussion by aaronh13** [photoprism/photoprism#2573] [Q&A]
On the sponsor page this is listed:

> Are there alternatives to a recurring subscription?
> For those who don't like recurring subscriptions, there will be discounted multi-year / lifetime payment options

Any ideas on when the lifetime plan will be available? 
(yes I know there are one time payment plans, but the support is only for 12m/24m and not truly lifetime).

thank you.

**Comment by graciousgrey** [comment]
Multi-year/lifetime payment options may be introduced once we start using Stripe. What would you be willing to pay and what do you expect in return?

**Reply by aaronh13** [reply]
I don't know about multi-year, but I think that $200-$250 would be a fair price for lifetime.

There is currently the $128 one time option for 24 months of sponsored features. The extra $70-$170 would equate to another 1-2 years essentially at the cost right now. I think that'd be a fair option. Essentially pay for 3-4 years upfront means lifetime.

Could also go the route to do this as a special pricing offer for a little while or for a limited number of users and then have the price set to $300-$350, occasionally giving out special discounts to the $200-$250 range for black Friday/Cyber Monday with coupon codes working in Stripe pretty easily.

Offering lifetime does give a boost to profits in the beginning, but overtime value is lost if everyone got a lifetime membership. Keeping it limited or raising the price with sales occasionally works to combat this, but overall the premium features, are really great, but also a bit limited. Adding something like multi-user support (allowing different users with different albums/private access) behind premium features would surely get more folks on board.

I think the only thing I'd expect is the sponsored features never expiring and 'getting help in community chat'.

**Reply by lastzero** [reply]
Thank you very much for your feedback! It is really helpful for us. By the way, we have started to update our website. You can now find a feature overview and additional pricing information at <https://photoprism.app/editions#compare>.

**Reply by PokeFoundry** [reply]
Just wanted to see if there had been any update on lifetimes this since it's been about a month.

Thank you.

**Reply by lastzero** [reply]
For this, we need Stripe, which has to be fully integrated and tested. We just got back from summer break and are now busy merging pull requests and implementing full multi-user support. We're working as fast as we can and will let you know when it's ready.

**Reply by polhaghverdian** [reply]
Hi,

Any more information regarding this question? I am also interested in a lifetime plan.

**Reply by lastzero** [reply]
A little more patience, please. We are considering to offer lifetime memberships in limited numbers, similar to Kickstarter. Not finally decided yet as we had too much other work.

**Reply by dmfarcas** [reply]
Very interested in the lifetime membership! I am using Photoprism to _get rid_ of my iCloud subscription. I'd like to redirect that money to this project since you're doing great work, but I just don't like subscriptions :))

**Reply by lastzero** [reply]
These are actual memberships, so we can build something together, increase your benefits over time and make sure you continue to get updates. Note that we have already released many member features to the public, so the features you get for free in the future are not the same as you get now.

Should you be happy with what you got, you can stop supporting us. This isn't possible with a cloud solution, as you will lose access to all your data. So the consequence of canceling your membership is completely different and not comparable.

That being said, we are working on a more affordable feature pack and you can automatically get a lifetime plus membership from the Silver tier up. See the Membership FAQ on our website for details.
